sholes/roster.json

```diff
@@ -12,7 +12,16 @@
     "students": [
       {
         "name": "Young Kim",
-        "github": "CKIMM7"
+        "github": "CKIMM7",
+        "materials": [
+          {
+            "type":"CV"
+          },
+          {
+            "type":"video",
+            "url": "https://youtu.be/dddrxqmlBJs"
+          }
```

Comment:
Can you please replace `"url": "https://youtu.be/dddrxqmlBJs"` with `"url": "dddrxqmlBJs"` :) ?